src/Symfony/Component/Workflow/Marking.php

```diff
@@ -27,23 +27,64 @@ class Marking
     public function __construct(array $representation = [])
     {
         foreach ($representation as $place => $nbToken) {
-            $this->mark($place);
+            $this->mark($place, $nbToken);
         }
     }
 
-    public function mark(string $place): void
+    /**
+     * @param int $nbToken
+     */
+    public function mark(string $place /* , int $nbToken = 1 */): void
     {
-        $this->places[$place] = 1;
+        if (1 === \func_num_args()) {
```

Comment:
please use `>` and a ternary operator, see maybe how we do it on 6.4 for inspiration

Reply:
mwai :| it's really a matter of personal taste :D . Anyway, updated

Reply:
consistency > personal taste ;)